agent/lib/kontena/workers/container_health_check_worker.rb

```diff
@@ -9,7 +9,7 @@ class ContainerHealthCheckWorker
     include Kontena::Helpers::PortHelper
     include Kontena::Helpers::RpcHelper
 
-    HEALTHY_STATUSES = [200]
+    HEALTHY_STATUSES = (200 .. 299).to_a.freeze
```

Comment:
Any particular reason to use `to_a` here? I just imagine that `Range#include?` is going to be more efficient than `Array#include?` with 100 elements, but it doesn't really matter.